meeting-minutes/2024-12-18.md

```diff
@@ -0,0 +1,134 @@
+# 1. Opening Activities
+
+## 1.1 Opening comments (Co-Chair)
+
+## 1.2 Introduction of participants/roll call (Co-Chair)
+
+Quorum requires participation of 9 or more of the 17 voting members (including the officers).
+
+| First Name | Last Name  | Company                                                     | Role(s)                   | Present |
+|:-----------|:-----------|:------------------------------------------------------------|:--------------------------|:--------|
+| Adrian     | Diglio     | Microsoft                                                   | Voting Member             | ??      |
+| David      | Kemp       | National Security Agency                                    | Member                    | ??      |
+| Denny      | Page       | Individual                                                  | Voting Member             | ???     |
+| Duncan     | Sparrell   | sFractal Consulting LLC                                     | Voting Member             | ??      |
+| Feng       | Cao        | Oracle                                                      | Member                    | ???     |
```

Comment:
Feng was present the last 2 meetings - so he could be voter; if he isn't a permanent non-voter.

Reply:
Fend should have voting rights, even if September meeting missed.